### ISSUE TITLE
tensorlayer Supports BIRNN with batch normal????

### ISSUE THREAD

**Issue by guoying1030** [zsdonghao/tensorlayer#80]
tensorlayer Supports BIRNN with batch normal????

**Comment by wagamamaz** [comment]
yeah, you can use `BatchNormLayer` and `BiDynamicRNNLayer` or `BiRNNLayer`

**Comment by guoying1030** [comment]
net = tl.layers.BiRNNLayer(net, cell_fn=tf.nn.rnn_cell.LSTMCell,
                                              n_hidden=hidden_size,n_steps=1000,
                                              initializer=tf.random_uniform_initializer(0, 0.01),
                                              name='DeepBiRnn_' + str(i),return_seq_2d=isReturn2d)
            print 'a'
            net = tl.layers.BatchNormLayer(net, act=lambda x: tl.act.ramp(x,0,20),name='BatchNormal_'+str(i))


?????

**Comment by guoying1030** [comment]
@wagamamaz ,,,net = tl.layers.BiRNNLayer(net, cell_fn=tf.nn.rnn_cell.LSTMCell,
                                              n_hidden=hidden_size,n_steps=1000,
                                              initializer=tf.random_uniform_initializer(0, 0.01),
                                              name='DeepBiRnn_' + str(i),return_seq_2d=isReturn2d)
            print 'a'
            net = tl.layers.BatchNormLayer(net, act=lambda x: tl.act.ramp(x,0,20),name='BatchNormal_'+str(i))


is this it???

**Comment by zsdonghao** [comment]
@guoying1030 yes
